src/Library/Environment/Win/WinEnvironment.cpp

```diff
@@ -4,7 +4,6 @@
 #include <Windows.h>
 
 #include <cstdlib>
-#include <string>
```

Comment:
I'm not sure about this X)

Reply:
I don't know how we ended up including twice the <string> header :D probably a bad auto-merge.